modules/Microsoft.Network/dnsForwardingRulesets/forwardingRules/deploy.bicep

```
@@ -0,0 +1,64 @@
+@description('Required. Name of the Forwarding Rule.')
+@minLength(1)
+param name string
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableDefaultTelemetry bool = true
+
+@description('Required. Name of the parant DNS Forwarding Ruleset.')
+param dnsForwardingRulesetName string
```

```suggestion
@description('Conditional. Name of the parent DNS Forwarding Ruleset. Required if the template is used in a standalone deployment.')
param dnsForwardingRulesetName string
```

Note: This requires an update of the readme too.